airbyte-server/src/main/java/io/airbyte/server/ConfigDumpExporter.java

```diff
@@ -56,20 +56,18 @@
  * would fail. 2. Unlike ArchiveHandler, this doesn't take the dump of specific files but looks at
  * the config directory and takes the full dump of whatever is available
  */
-public class ConfigDumpExport {
+public class ConfigDumpExporter {
```

Comment:
Refactored this class so the constructor takes in the dbs so that it can be dependency injected, but then `dump` can be called many times.